doc(data-handling-overview): Update description and title (#962)
bonitasoft/bonita-doc: md/data-handling-overview.md

# Data handling overview

> How to effectively use the different kinds of data in the Bonita application

Reviewer: We usually talk about Bonita Platform in the rest of the documentation. So I would change the end of the sentence. What do you think of : "How to effectively use the different kinds of data available within Bonita Platform" ?

Author: I agree it should be consistent with other docs

> How to effectively use the different kinds of data in the Bonita application

Author: ```suggestion
How to effectively use the different kinds of data available within Bonita Platform
```
